examples/express/client/src/App.tsx

```diff
@@ -0,0 +1,104 @@
+import { generateComponents } from "@uploadthing/react";
+
+import type { OurFileRouter } from "../../uploadthing";
+
+const { UploadButton, UploadDropzone } = generateComponents<OurFileRouter>();
+
+export function App() {
```

Comment:
again, i feel like this is too much duplication.

i think we should rethink our examples. every single examples doesn't have to show all use cases, just have minimal ones for setting up the file route and serving it, and consuming a single endpoint would be fine.

then we can have specific examples for different use cases (preferably using next.js???). that way it's way more easily browsed for users coming to our repo looking for an example on how to do certain things.

tldr; more, contrived examples rather than a few bigger ones provides a better use experience. i was actually thinking of doing this but then i went on vacation. might get to it later today or early next week.

Reply:
Your point makes sense. In addition, considering your other comments, it seems we need a PR with many different improvements to docs and examples. Or separate ones for these two items

Personally, I had created example as development playground in the first place. I simply copied filerouter from other example and bootstrapped app just to be able to test new stuff